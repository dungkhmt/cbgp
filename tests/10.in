20 20
20 36
20 11
11 2
2 3
3 9
9 12
12 8
8 20
5 13
13 4
4 16
16 15
15 7
7 5
6 19
19 1
1 14
14 18
18 6
18 7
6 1
17 10
15 9
10 14
12 15
16 9
13 19
5 8
20 2
5 6
17 14
16 3
12 7
10 18
4 2
20 13
4 1
1
0 0 20 2 21 3 22 0 11 3 8 2 18 10 16 6 22 4 14 14 17 2 18 8 13 1 12 12 19 9 28 0 14 13 17 11 9 1 18 4 
