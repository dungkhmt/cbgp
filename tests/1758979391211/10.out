(19, 10), (10, 1), (1, 2), (2, 8), (8, 11), (11, 7), (7, 19), (4, 12), (12, 3), (3, 15), (15, 14), (14, 6), (6, 4), (5, 18), (18, 0), (0, 13), (13, 17), (17, 5), (17, 6), (5, 0), (16, 9), (14, 8), (9, 13), (11, 14), (15, 8), (12, 18), (4, 7), (19, 1), (4, 5), (16, 13), (15, 2), (11, 6), (9, 17), (3, 1), (19, 12), (3, 0), 
numberIntersection=0 
Var(nodePos)=3031.7375 
minAngle=0.0 
minDistance=0.0 
Var(edgeLength)=1376.284720432932 
sumAngle=6.540388037598069 
sumDistance=36416.896103320956 
minEdgeLength=1.0 
sumEdgeLength=1666.9745076862096 

0.0000000000 3031.7375000000 0.0000000000 0.0000000000 1376.2847204329 6.5403880376 36416.8961033210 1.0000000000 1666.9745076862

0: (0, 0),
1: (77, 1),
2: (78, 1),
3: (79, 0),
4: (2, 2),
5: (1, 78),
6: (2, 79),
7: (78, 78),
8: (79, 1),
9: (79, 80),
10: (78, 77),
11: (79, 79),
12: (1, 1),
13: (0, 80),
14: (80, 80),
15: (80, 0),
16: (1, 80),
17: (1, 79),
18: (1, 2),
19: (2, 1),

Time taken: 38.257 s
0 0 77 1 78 1 79 0 2 2 1 78 2 79 78 78 79 1 79 80 78 77 79 79 1 1 0 80 80 80 80 0 1 80 1 79 1 2 2 1 
0: (0, 0), 1: (77, 1), 2: (78, 1), 3: (79, 0), 4: (2, 2), 5: (1, 78), 6: (2, 79), 7: (78, 78), 8: (79, 1), 9: (79, 80), 10: (78, 77), 11: (79, 79), 12: (1, 1), 13: (0, 80), 14: (80, 80), 15: (80, 0), 16: (1, 80), 17: (1, 79), 18: (1, 2), 19: (2, 1), 
numberIntersection=0 minAngle=0.0 sumAngle=6.54038803758112 minEdgeLength=1.0 sumEdgeLength=1666.974507686209 minDistance=0.0 sumDistance=36416.89610332071 
